e2e/index.js

```diff
@@ -26,7 +26,10 @@ function createConfig(pluginConfig) {
       path: outputPath,
       filename: `tmp.js`
     },
-    plugins: [new HookShellScriptPlugin(pluginConfig)]
+    plugins: [new HookShellScriptPlugin(pluginConfig)],
+    infrastructureLogging: {
+      level: "none"
+    }
```

Comment:
Infrastructure log messaging is verified in the other tests, and doesn't need to be emitted here.